src/components/PeoplePage.tsx

```diff
@@ -1,29 +1,67 @@
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { useEffect, useState } from 'react';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { useLocation, useSearchParams } from 'react-router-dom';
 
 export const PeoplePage = () => {
+  const [searchParam, setSearchParam] = useSearchParams();
+  const sex = searchParam.get('sex') || '';
+  const nameSearch = searchParam.get('name') || '';
+  const centuries = searchParam.getAll('centuries') || [];
+  const sort = searchParam.get('sort') || '';
```

Comment:
The `sort` parameter is retrieved from the search parameters but is not used anywhere in the component. Ensure that sorting functionality is implemented as per the task requirements, which involves sorting by `name`, `sex`, `born`, and `died` fields.